cmake/configs/default.cmake

```diff
@@ -317,7 +317,7 @@ hunter_config(xext VERSION 1.3.1)
 hunter_config(xextproto VERSION 7.2.1)
 hunter_config(xf86vidmodeproto VERSION 2.3.1)
 hunter_config(xfixes VERSION 5.0.1)
-hunter_config(xgboost VERSION 0.40-p9)
+hunter_config(xgboost VERSION 0.7.0-p0)
```

Comment:
0.40 to 0.7? Is it downgrade?

Reply:
I guess they have changed the versioning scheme to a strict semantic versioning:

https://github.com/dmlc/xgboost/issues/2925

> regularly publish minor updates, e.g. 0.7.1, 0.7.2, and so forth. Minor updates shall accommodate bug fixes and minor changes.

As you point out, the previous release used a slightly different versioning scheme:

https://github.com/dmlc/xgboost/releases

v0.40, v0.60

Reply:
So in short 0.7 is newer than 0.40.

Reply:
Yes.